driver/pace/driver/comm.py

```diff
@@ -66,7 +78,7 @@ def from_dict(cls, config: dict):
         )
 
 
-@CommConfig.register("mpi")
+@CreatesCommSelector.register("mpi")
```

Comment:
I think this naming is much clearer since we have so many configs everywhere already.